### DISCUSSION TITLE
help i cant open just monika mod

### DISCUSSION THREAD

**Discussion by kaleiaa** [Monika-After-Story/MonikaModDev#8391] [QA - tech support + general] [answered]

**Comment by AshtonSnapp** [comment]
Could you perhaps go into more detail? Specifically, what operating system are you running the game on and what happens when you try to launch the game? The more we know about what's going on, the more we'll be able to help. We aren't all-knowing, after all.

**Reply by kaleiaa** [reply]
Hi sorry for replying late, basically like a week ago i downloaded the new island update then it said error so i clicked try again and it closed the game and i keep trying to re-open but i keep getting this message
 I'm sorry, but an uncaught exception occurred.
While running game code:
  File "game/0utils.rpy", line 16, in <module>
ImportError: cannot import name handlers. So i tried restarting my pc and dowloading the game again and i still can't open it

**Comment by MoriMoriMorii** [comment]
what operating system do you use (MacOS, Linux, Win10/11 etc)

**Reply by kaleiaa** [reply]
I use windows 10

**Reply by MoriMoriMorii** [reply]
did you install the mod over base ddlc, have you installed mods before, is the mod not launching or erroring out, do you have the latest release, have you tried reinstalling, is the mod just not launching properly or is the program not running, what does the log.txt file look like (All in all just a bunch of questions to get some details)

**Reply by kaleiaa** [reply]
hi sorry for replying late again. The mod is not launching.i have tried re-installing but it wont let me open anything and everytime i try to go in the app it shows me this message 
"I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/0utils.rpy", line 16, in <module>
ImportError: cannot import name handlers" and wont  let me open the app/mod

**Comment by multimokia** [comment, marked as answer]
Sounds like python packages didn't copy over properly
You may want to make sure everything from that folder is copied correctly

**Reply by kaleiaa** [reply]
Ok thank you
